aws/resource_aws_spot_fleet_request_test.go

```diff
@@ -2330,3 +2373,23 @@ resource "aws_spot_fleet_request" "test" {
 }
 `, rName, validUntil)
 }
+
+func testAccAWSSpotFleetRequestZeroCapacityConfig(rName string, rInt int, validUntil string) string {
+	return testAccAWSSpotFleetRequestConfigBase(rName, rInt) + fmt.Sprintf(`
+resource "aws_spot_fleet_request" "test" {
+    iam_fleet_role = "${aws_iam_role.test-role.arn}"
+    spot_price = "0.005"
```

Comment:
See #14083, #13091; `0.005` is not a valid Gov Cloud value.

```suggestion
    spot_price = "0.05"
```